src/platformIcon.tsx

```diff
@@ -0,0 +1,167 @@
+import React from "react";
+import styled from "@emotion/styled";
+
+const PLATFORM_TO_ICON = {
+  android: "android",
+  apple: "apple",
+  cocoa: "apple",
+  "cocoa-objc": "apple",
+  "cocoa-swift": "swift",
+  cordova: "cordova",
+  csharp: "csharp",
+  "csharp-aspnetcore": "dotnet",
+  dart: "dart",
+  default: "default",
+  dotnet: "dotnet",
+  electron: "electron",
+  elixir: "elixir",
+  flutter: "flutter",
+  go: "go",
+  java: "java",
+  "java-android": "android",
+  "java-appengine": "app-engine",
+  "java-log4j": "java",
+  "java-log4j2": "java",
+  "java-logback": "logback",
+  "java-logging": "java",
+  javascript: "javascript",
+  "javascript-angular": "angularjs",
+  "javascript-angularjs": "angularjs",
+  "javascript-backbone": "backbone",
+  "javascript-browser": "javascript",
+  "javascript-ember": "ember",
+  "javascript-react": "react",
+  "javascript-vue": "vue",
+  native: "nativec",
+  node: "nodejs",
+  "node-awslambda": "aws",
+  "node-azurefunctions": "azure",
+  "node-connect": "nodejs",
+  "node-express": "express",
+  "node-gcpfunctions": "gcp",
+  "node-koa": "koa",
+  "objective-c": "apple",
+  perl: "perl",
+  php: "php",
+  "php-laravel": "laravel",
+  "php-monolog": "php",
+  "php-symfony2": "symfony",
+  "php-symfony": "symfony",
+  python: "python",
+  "python-aiohttp": "aiohttp",
+  "python-awslambda": "aws",
+  "python-azurefunctions": "azure",
+  "python-bottle": "bottle",
+  "python-celery": "celery",
+  "python-django": "django",
+  "python-falcon": "falcon",
+  "python-flask": "flask",
+  "python-gcpfunctions": "gcp",
+  "python-pylons": "python",
+  "python-pyramid": "pyramid",
+  "python-pythonawslambda": "aws",
+  "python-rq": "redis",
+  "python-sanic": "python",
+  "python-tornado": "tornado",
+  "python-tryton": "tryton",
+  "react-native": "react",
+  ruby: "ruby",
+  "ruby-rack": "ruby",
+  "ruby-rails": "rails",
+  rust: "rust",
+  swift: "swift",
+  // Don't add new platforms down here!
+  // Please add them where they belong alphabetically
+};
+
+function normalizePlatform(platform: string): string {
+  // sentry uses format python-django, but docs uses python.django
+  // this function normalizes that
+  return platform.replace(".", "-");
+}
+
+function getIcon(platform: string): Platform {
+  const normalizedPlatform = normalizePlatform(platform);
+  const icon = PLATFORM_TO_ICON[normalizedPlatform];
+
+  if (icon) {
+    return icon;
+  }
+
+  if (normalizedPlatform.includes("-")) {
+    return getLanguageIcon(normalizedPlatform);
+  }
+
+  return "default";
+}
+
+function getLanguageIcon(platform: string): Platform {
+  const [language] = normalizePlatform(platform).split("-");
+
+  return getIcon(language);
+}
+
+type Platform = keyof typeof PLATFORM_TO_ICON;
+
+type Props = React.HTMLAttributes<HTMLDivElement | HTMLImageElement> & {
+  platform: string;
+  size?: string | number;
+  format?: "sm" | "lg";
```

Comment:
Good call separating size/format props

Reply:
I'm not sure if the `format` name fits here. I'd usually think of ".svg, .png, .jpg, etc" .. I also noticed that in sentry we usually call "sm" | "lg", etc `size`.. maybe we could find a better name ?

Reply:
This prop basically tells if we should use the small format of logos (used in dropdowns, small badges, etc.):
![image](https://user-images.githubusercontent.com/9060071/93436333-ee1a3a80-f8ca-11ea-8223-4f383ecda1fe.png)
or the large one (used in project creation, docs, etc.):
![image](https://user-images.githubusercontent.com/9060071/93436427-13a74400-f8cb-11ea-9793-6dcac7b2e4e1.png)

Do you have a suggestion for a better name?

Reply:
maybe "layout"?